Intro to open source with GitHub/README.md

```diff
@@ -0,0 +1,22 @@
+# How to work on opensource contribution with Git
+
+## Making a contribution
+1. Fork the repo which you want to work on (eg. https://github.com/take2rohit/sih_number_plate is original repo where you want to contribute) 
+1. Clone the forked repository (not the above repo link)
+`git clone https://github.com/your_userid/sih_number_plate`
+3. Do your changes whatever you wish and commit 
+`git add .`
```

Comment:
Some new people might not know. Maybe we can replace with '`git add file1 file2 file3` to add particular files and `git add .` to add all files'. Also, they have to do in the root folder of the repo (is it always clear?).

Reply:
If someone is contributing its expected he should know git add